<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addmember</class>
 <widget class="QMainWindow" name="addmember">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="11" column="0">
       <widget class="QLineEdit" name="lineEdit_4"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="styleSheet">
         <string notr="true">QLabel{
	font: 600 14pt &quot;Zilla Slab SemiBold&quot;;
	color: rgb(0, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>First Name</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">QLabel{
	font: 600 14pt &quot;Zilla Slab SemiBold&quot;;
	color: rgb(0, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">QLabel{
	font: 600 14pt &quot;Zilla Slab SemiBold&quot;;
	color: rgb(0, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>Last Name</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLineEdit" name="lineEdit_5"/>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="lineEdit_7"/>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_9">
        <property name="styleSheet">
         <string notr="true">QLabel{
	font: 600 14pt &quot;Zilla Slab SemiBold&quot;;
	color: rgb(0, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>Phone Number</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">QLabel{
	font: 600 14pt &quot;Zilla Slab SemiBold&quot;;
	color: rgb(0, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>Gender</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEdit_6"/>
      </item>
      <item row="7" column="0">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>Male</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Female</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>No gender</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item alignment="Qt::AlignVCenter">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_4">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(226, 226, 226);
	color: rgb(0, 0, 0);
	font: 700 11pt &quot;Segoe UI&quot;;
}</string>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_3">
         <property name="styleSheet">
          <string notr="true">QPushButton{
	background-color: rgb(226, 226, 226);
	color: rgb(0, 0, 0);
	font: 700 11pt &quot;Segoe UI&quot;;
}</string>
         </property>
         <property name="text">
          <string>Add Member</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>422</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
